.github/workflows/microbenchmarks-ci.yaml

```diff
@@ -14,6 +14,10 @@ env:
   HEAD: ${{ github.event.pull_request.head.sha }}
   BUCKET: "cockroach-microbench-ci"
   PACKAGE: "pkg/sql/tests"
+  # Base offset: increases the base commit selection by the given offset, so
+  # benchmarks are able to run against commits in the PR. Use only for testing;
+  # always reset to default (0) before merging.
```

Comment:
Sounds prone to honest mistakes. Should we add a github action check to prevent merging if this file doesn't contain `BASE_OFFSET: 0`?